src/v4/scripts/process-legacy-core.js

```diff
@@ -0,0 +1,54 @@
+const dependencyTree = require('dependency-tree');
+const glob = require('glob');
+const Runner = require('jscodeshift/src/Runner');
+const path = require('path');
+const fs = require('fs-extra');
+const { execSync } = require('child_process');
+const rimraf = require('rimraf');
+const cpx = require('cpx');
+
+const coreDir = 'src/v4/core';
+const v4Path = path.resolve(__dirname, '../');
+
+/** @type {{ [coreImport: string]: string[]; }} */
+const deps = {};
+
+// Copy core src
+rimraf.sync(`${v4Path}/core`);
+rimraf.sync('temp');
+execSync('git clone https://github.com/dojo/framework.git temp --branch v3.0.1 --single-branch --depth 1');
+fs.renameSync('temp/src/core', `${v4Path}/core`);
+rimraf.sync('temp');
+
+// Generate dependency list
+const paths = glob.sync(`${coreDir}/**/*.ts`);
+paths.forEach((filename) => {
+	const list = dependencyTree.toList({
```

Comment:
Sorry to be nitpicky, but the JS files seem to have inconsistent spacing. They should be tabs.